scipy/stats/tests/test_distributions.py

```diff
@@ -1434,14 +1434,18 @@ def test_sf(self):
         expected = (scale/x)**b   # 2.25e-18
         assert_allclose(p, expected)
 
+    @pytest.fixture(scope='class')
```

Comment:
Fine, but note that this will set once the generator for the test class. To debug and have the same state in a function, we have to run all tests of the class. Not individual tests, for that we would need `scope='function'` (the default).

Reply:
Nice catch. I think it should be `function`.

Reply:
Let's do this then and I can merge if ready on your side. @mdhaber fine with you as well?

```suggestion
    @pytest.fixture(scope='function')
```